MemoryManagement/src/memorymanagementAlgorithm/Process.java

```diff
@@ -1,140 +1,192 @@
 package memorymanagementAlgorithm;
 
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+/**
+ *
+ * @author ahmed
+ */
+
 import java.util.*;
 import javafx.scene.paint.Color;
+//Vector<Double> list = new Vector<Double>();
 
 public class Process {
 
+    
+
+    private Color color;
     private static int Process_ID = 0;
     private int ID;
-    private Color color;
     private boolean inserted = false;
+    private int number_of_segments;

```

Comment:
this is a stupid redundant. every vector has its own size and its complicity is O(1). So you recount after him!

Reply:
this was a basic algorthim (naive) need to be stupid .
you are right fixed.